TransformRecorder/qSlicerTrackedSequenceBrowserWriter.cxx

```diff
@@ -315,14 +314,30 @@ ::writeSQBR( vtkMRMLSequenceBrowserNode* trackedSequenceBrowserNode, std::string
     {
       continue;
     }
-    vtkSmartPointer< vtkMRMLNode > currTempProxyNode = currProxyNode->CreateNodeInstance();
+    vtkSmartPointer<vtkMRMLNode> currTempProxyNode = vtkSmartPointer<vtkMRMLNode>::Take(currProxyNode->CreateNodeInstance());
     currTempProxyNode->Copy( currProxyNode );
     currTempProxyNode->SetScene( tempScene.GetPointer() );
     tempScene->AddNode( currTempProxyNode );
     tempTrackedSequenceBrowserNode->AddProxyNode( currTempProxyNode, currTempSequenceNode, false );
 
     this->copyNodeAttributes( currProxyNode, currTempProxyNode );
     currTempProxyNode->UpdateReferences();
+
+    if (currSequenceNode->GetStorageNode())
+    {
+      // Use the same class of sequence storage node that is used in the scene
+
+      vtkSmartPointer<vtkMRMLStorageNode> currSequenceStorageNode = currSequenceNode->GetStorageNode();
+      vtkSmartPointer<vtkMRMLStorageNode> currTempSequenceStorageNode = vtkSmartPointer<vtkMRMLStorageNode>::Take(vtkMRMLStorageNode::SafeDownCast(currSequenceStorageNode->CreateNodeInstance()));
+      currTempSequenceStorageNode->Copy(currSequenceStorageNode);
```

Comment:
This copies node name, references, etc. Shouldn't we just use CopyContents?

Reply:
Probably, but the vtkMRMLStorageNode class doesn't have a CopyContent method. Compression parameters are only copied in vtkMRMLStorageNode::Copy.

I assume that they should be moved to CopyContent?

Reply:
It would be probably cleaner to add a CopyContent method for the storage node because it is very rare that you actually want to to copy the node name, singleton tag, and especially references. I hope it won't make storage nodes appear in Sequences module as potential proxy nodes (if they appear there then we should figure out something for hiding them).

Anyway, this is just a small opportunity to make the design a bit more consistent (avoid use of `Copy` method) and if you run into any complications then you can leave things as is.

Reply:
Ok, Thanks! I'll add the CopyContent method and come back to this. For now I will integrate as is so that PerkTutor can use compressed video when saving.